src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/highRiskBreachData.tsx

```diff
@@ -42,12 +42,12 @@ export type HighRiskBreach = {
 function getHighRiskBreachesByType({
   dataType,
   breaches,
+  l10n,
 }: {
-  dataType: string;
+  dataType: HighRiskBreachTypes;
   breaches: GuidedExperienceBreaches;
+  l10n: ExtendedReactLocalization;
 }) {
-  const l10n = getL10n();
-
   // TODO: Expose email list & count here https://mozilla-hub.atlassian.net/browse/MNTOR-2112
```

Comment:
Thanks to you it seems like the TODO regarding adding the count is resolved. :)

Reply:
As discussed on Slack, I'm not actually sure what this TODO is and whether I've resolved it, so leaving for now :)